com.unity.render-pipelines.universal/Editor/Deprecated.cs

```diff
@@ -6,3 +10,16 @@ internal class Deprecated
     {
     }
 }
+
+namespace UnityEditor.Rendering.Universal
+{
+    [Obsolete("ForwardRendererDataEditor has been deprecated. Use StandardRendererDataEditor instead (UnityUpgradable) -> StandardRendererDataEditor", true)]
+    [MovedFrom("UnityEditor.Rendering.LWRP")]
```

Comment:
please test this with a real project that has references to `UnityEditor.Rendering.LWRP.ForwardRendererDataEditor`
and make sure it get updated to `StandardRendererDataEditor`

Reply:
I created a template LWRP project with 2019.2.0f1, which is using LWRP 6.9.0 and added a script file referencing `UnityEditor.Rendering.LWRP.ForwardRendererDataEditor`.

Unfortunately, this CANNOT be upgraded directly to latest official URP package using the editor from trunk: it will fail to rename `UnityEditor.Rendering.LWRP.ForwardRendererDataEditor` into `UnityEditor.Rendering.UniversalRP.ForwardRendererDataEditor`. So it will also not work for my branch.

However, the project can be upgraded to UniversalRP 7.3.1 using editor 2019.3.15f1.
Then, using the Editor in trunk with the URP package from my branch, `UnityEditor.Rendering.UniversalRP.ForwardRendererDataEditor` is successfully renamed  into `UnityEditor.Rendering.UniversalRP.StandardRendererDataEditor`.

Reply:
It looks like the upgrading feature from LWRP to latest URP is already broken ...